packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -1600,27 +1605,22 @@ class ApiGateway {
   protected liveness: RequestHandler = async (req, res) => {
     let health: 'HEALTH' | 'DOWN' = 'HEALTH';
 
+    const handleError = (e: any) => {
```

Comment:
Why do you need this wrapper function instead of using the existing method?

Reply:
Just DRY. What existing method do you mean?

Reply:
Well, you created a `handleProbeError` that does all stuff. And instead of using it and just reassigning the `health` variable, you create another function, close this variable inside, and call this function to assign the variable. In my opinion, this makes code less readable and more buggy. Usually, we try to avoid closures if it is possible. And here, I don't see any significant reason for this. Does it make sense to you?